src/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './constants';
 export * as Crypto from './crypto';
 export * from './utils';
+export * from './utils/types';
```

Comment:
I'd move this file to `./types` instead to better match with the rest of our projects

```suggestion
export * from './types';
```